feat(docs): Note for path-style-access with Minio (#1500)
spinnaker/spinnaker.github.io: setup/install/storage/minio.md

hal config storage edit --type s3
```

## Notes

> > :warning: Unless Minio is configured to [support virtual-host-style requests](https://docs.min.io/docs/minio-server-configuration-guide.html#Domain), the path style access mode will have to be enabled: `hal config storage s3 edit --path-style-access true`. Front50 will otherwise refuse to start and throw `java.net.UnknownHostException: $BUCKET_NAME.localhost` exceptions.

Reviewer: ```suggestion
> :warning: Unless you've configured Minio to [support virtual-host-style requests](https://docs.min.io/docs/minio-server-configuration-guide.html#Domain), you need to enable the path-style access mode: `hal config storage s3 edit --path-style-access true`. Otherwise, Front50 will fail with `java.net.UnknownHostException: $BUCKET_NAME.localhost` exceptions.
```
